docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/bitwise/bitwise-left-shift-15.rst

```diff
@@ -0,0 +1,112 @@
+.. {#openvino_docs_ops_bitwise_BitwiseLeftShift_15}
+
+BitwiseLeftShift
+================
+
+
+.. meta::
+  :description: Learn about BitwiseLeftShift-15 - an element-wise, bitwise left shift operation.
+
+**Versioned name**: *BitwiseLeftShift-15*
+
+**Category**: *Bitwise binary*
+
+**Short description**: *BitwiseLeftShift* performs a bitwise left shift operation operation, applying multi-directional broadcast rules.
+
+**Detailed description**: Before performing the operation, input tensors *a* and *b* are broadcasted if their shapes are different and the ``auto_broadcast`` attribute is not ``none``. Broadcasting is performed according to the ``auto_broadcast`` value.
+
+After broadcasting input tensors *a* and *b*, *BitwiseLeftShift* performs a bitwise left shift operation for each corresponding element in the given tensors, based on the following equation:
+
+.. math::
+
+   o_{i} = a_{i} \ll b_{i}
+
+
+.. note::
+
+    If the number of shifts is negative or exceeds the total number of bits in the type **T**, the behaviour can be undefined or implementation defined (depends on the hardware).
```

Comment:
I think exceeds or equals (as well) because it is not clear what will be for signed integers in case of equal case. Please check

Reply:
Agree